Userland/Libraries/LibCpp/AST.cpp

```diff
@@ -572,6 +572,24 @@ StringView TemplatizedName::full_name() const
     return *m_full_name;
 }
 
+void SizedName::dump(FILE* output, size_t indent) const
+{
+    Name::dump(output, indent);
+    print_indent(output, indent + 1);
+
+    StringBuilder dimInfo;
```

Comment:
Camel case please: `dim_info`

Reply:
Also, no abbreviations in variable names :)

Reply:
Chose a better name and removed camel case